recipes/tblis/build.sh

```diff
@@ -0,0 +1,7 @@
+CFLAGS=$(echo "${CFLAGS}" | sed "s/-march=[a-zA-Z0-9]*//g")
```

Comment:
Why are you removing this?

Reply:
TBLIS builds optimized microkernels for a wide range of x86 processors and needs to pass in specific -march flags to compile properly. The default conda-forge compile flags `-march=nocona -mtune=haswell` will interfere with this.

TBLIS detects the user's architecture at runtime and selects the best codepath. Therefore, no illegal instruction errors should occur. The conda-forge recipe for blis [also uses this line](https://github.com/conda-forge/blis-feedstock/blob/main/recipe/build.sh).